source/atlas-cli-changelog.txt

```diff
@@ -12,6 +12,82 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.1.5:
+
+{+atlas-cli+} 1.1.5
+-------------------
+
+Released 2 August 2022
+
+- Fixes a bug where telemetry tracking events were not propagated when 
+  a default profile was not set.
+
+- Fixes a bug where it was not possible to 
+  filter organizations or projects by name during onboarding.
+
+- Fixes a bug where some options were not validated on command 
+  :ref:`atlas-backups-restores` job start.
+
+- Improves documentation and adds examples to 
+  :ref:`atlas-privateEndpoints-aws-describe`.
+
+- Improves onboarding for users with more than 500 organizations.
+
+- Improves the message when an operation completes in a free 
+  cluster, directing users to an upgrade command.
+
+- Updates the help message for roles in :ref:`atlas-dbusers-create` 
+  command.
+
+- Adds "Upgrade One Shared Tier Cluster" endpoint to Atlas Go Client.
+
+- Adds new ``atlas projects`` commands:
+
+  - :ref:`atlas-projects-settings-update` updates settings of a 
+    given project.
+
+  - :ref:`atlas-projects-settings-describe` lists project settings.
+
+- Adds new ``atlas clusters`` commands:
+
+  - :ref:`atlas-clusters-advancedSettings-describe` gets advanced 
+    settings for a cluster.
+
+  - :ref:`atlas-clusters-upgrade` upgrades a shared cluster to a 
+    dedicated one.
+
+- Adds new ``atlas backups`` commands:
+
+  - :ref:`atlas-backups-schedule-delete` deletes all backup schedules 
+    for a given cluster.
+
+  - :ref:`atlas-backups-schedule-update` updates the backup schedule of 
+    a cluster.
+
+  - :ref:`atlas-backups-schedule-describe` gets the backup schedule of 
+    a cluster.
+
+  - :ref:`atlas-backups-restores-describe`
```

Comment:
Did you mean to have a description for this, or to remove it?

Reply:
No, I just didn't add one if it wasn't in the notes. Adding now :)